preset/README.md

```diff
@@ -84,11 +84,11 @@ docker run -it --rm \
 2. Select a test from the `sample-tests` folder and run it using the following command as an example:
 
     ```bash
-    bash sample-tests/onnx/run.sh
-    # or if no bash script is found
     python sample-tests/intel_extension_for_tensorflow/test_itex.py
     ```
 
+    Note: The `sample-tests` folder may differ in each container, and some tests use a bash script.
```

Comment:
It's better to use the alerts syntax for github markdown: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Reply:
Thanks for the feedback.
I made the change